legend-engine-ide-lsp-default-extensions/pom.xml

```diff
@@ -51,6 +51,11 @@
                 <scope>import</scope>
                 <version>${legend.engine.version}</version>
             </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.13</version>
+            </dependency>
```

Comment:
Intellij complains about the version engine imports (4.13.1), and I found is a bug on those patch versions.  SO suggested to just use 4.13.